### PR TITLE
getting and setting the minpoly

### PR DESCRIPTION
**Pull request by tthsqe12**
Would like @hannes14 to have a look at this before merging. It seems to work and is almost verbatim from the singular source code, but I don't know what I could have messed up here. I couldn't find a function for doing this in the kernel - maybe it is time for such a function?

### CODE REVIEW

File: coeffs.cpp [hunk 1]
```diff
@@ -39,6 +39,93 @@ void singular_define_coeffs(jlcxx::Module & Singular)
...
+            poly n = DEN((fraction)(p));
+            if (!p_IsConstantPoly(n, cf->extRing))
+            {
+                WarnS("denominator must be constant - ignoring it");
+            }
+            p_Delete(&n, cf->extRing);
+            DEN((fraction)(p)) = NULL;
+        }
+
+        if (redefine_from_algext)
```
Comment (hannes14) [inline coeffs.cpp:87]: redefine_from_algext cannot be true, as n_trasnExt is checked in the first line

Comment (hannes14) [reply coeffs.cpp:87]: see also the jjSetMinpoly, which was reworked and simplified in 
https://github.com/Singular/Singular/commit/311d0993aaf7a37ae5134ab3b58a01291ecdfb59

Comment (tthsqe12) [reply coeffs.cpp:87]: There is a
` BOOLEAN redefine_from_algext = FALSE;`
a few lines up to copy the original as much as possible.
So would this be good enough until we can get the next version of singular with new jjSetMinpoly?

Comment (hannes14) [reply coeffs.cpp:87]: fine for the moment.

**Comment by tthsqe12** [comment]
@thofma I am happy with this.

**Pull request merged**